---
layout: page
title:  李海廓课题组<br>肾脏与组学实验室
use-site-title: False
bigimg:
 - "/img/coverphotos/share-seq-umap.jpg" : "人类肾脏结构的单细胞多组学图谱 Single-cell multimodal atlas for human kidneys (Cell Metabolism 2024)"
 - "/img/coverphotos/kidney-maldi.jpg" : "人类肾脏结构的空间代谢组 Spatial metabolomics underlying regional kidney anatomy (Cell Metabolism 2024)"
 - "/img/coverphotos/sci-seq-injury.jpg" : "近端小管损伤型细胞态 Injured cell states of the proximal tubule (Cell Metabolism 2022；左图被选为NIDDK年度报告唯一插图)"
 - "/img/coverphotos/sci-seq-plin2.jpg" : "PLIN2是一类损伤型肾小管细胞态的脂滴标志物 PLIN2 is a lipid droplet marker in Type 1 Injured PT cells (Cell Metabolism 2022)"
 - "/img/coverphotos/sci-seq-lipid.jpg" : "脂质聚集是肾脏损伤和纤维化早期的代谢特征 Lipid accumulation in the acute phase of kidney injury (Cell Metabolism 2022)"
 - "/img/coverphotos/cover-spatial-omics.jpg" : "前沿空间表观遗传组和转录组技术的开发与应用 Development of spatially resolved epigenomics & transcriptomics coprofiling methods (Nature Protocols 2025)"


---
<hr>

<div style="overflow: auto;">
  <div style="float: left; width: 50%; padding-right: 20px; font-size: 16px;">
    <strong>Kidney injury and fibrosis</strong><br><br>
	  Kidney fibrosis represents the final common pathway of all chronic kidney diseases (CKD). Key challenges in addressing kidney fibrosis and halting CKD progression include understanding how to effectively target the health-to-disease transition of tubular epithelial cells and discovering strategies to promote cell redifferentiation and regeneration following kidney injury.<br><br>
	  Our research integrates human clinical resources, mouse models of kidney injury and fibrosis, tissue culture systems, real-time metabolic measurements, multiomics and bioinformatics approaches to uncover the genetic and epigenetic mechanisms driving kidney diseases.<br><br>
  </div>
  <div style="float: right; width: 50%;">
    <img src="/img/coverphotos/disease-study.jpeg" alt="disease-study.jpeg" style="width: 100%; max-width: 400px;" /><br>
  </div>
</div>

<hr>

<div style="overflow: auto;">
  <div style="float: left; width: 50%; padding-right: 20px; font-size: 16px;">
    <strong>Human diseases at the single-cell resolution</strong><br><br>
	  We have deep expertise in development and applications of various single-cell technologies.
	  Beyond the well-known droplet microfluidics approach, we employ the split-pool barcoding method, which offers unique advantages in ultra-high throughput, robust sample multiplexing and cost efficiency.<br><br>
	 The creation of the kidney single-cell transcriptomics reference atlas (Human Cell Atlas v1) marked a significant milestone. 
	  Now, we stand at the forefront of a transformative era, poised to develop a more comprehensive atlas — one characterized by greater coverage, a broader spectrum of disease subtypes, and enhanced multimodal complexity (HCA v2).<br><br>
  </div>
  <div style="float: right; width: 50%;">
    <img src="/img/coverphotos/sc-methods.jpeg" alt="sc-methods.jpeg" style="width: 100%; max-width: 400px;" /><br>
  </div>
</div>

<hr>

<div style="overflow: auto;">
  <div style="float: left; width: 50%; padding-right: 20px; font-size: 16px;">
    <strong>Spatially resolved multiomics</strong><br><br>
	  Never before have we had the ability to observe a tissue section at such a comprehensive molecular level and high resolution as we do today. The advent of spatial multiomics offers us invaluable opportunities to deepen our understanding of the structural changes in tissue organization and alterations in cell communication mechanisms underlying kidney physiology and disease states.<br><br>
	  We are using cutting-edge Deterministic Barcoding in Tissue (DBiT) approaches to achieve near-single-cell resolution and whole-genome coverage for exploring spatial transcriptomics and spatial epigenomics. Furthermore, we aim to develop methods to enable spatial multimodal profiling and integrative analysis.<br><br>
  </div>
  <div style="float: right; width: 50%;">
    <img src="/img/coverphotos/spatial-omics.jpeg" alt="spatial-omics.jpeg" style="width: 100%; max-width: 400px;" /><br>
  </div>
</div>


<div class="posts-list">
  {% for post in paginator.posts %}
  <article class="post-preview">
    <a href="{{ post.url | relative_url }}">
	  <h2 class="post-title">{{ post.title }}</h2>

	  {% if post.subtitle %}
	  <h3 class="post-subtitle">
	    {{ post.subtitle }}
	  </h3>
	  {% endif %}
    </a>

    <p class="post-meta">
      Posted on {{ post.date | date: site.date_format }}
    </p>

    <div class="post-entry-container">
      {% if post.image %}
      <div class="post-image">
        <a href="{{ post.url | relative_url }}">
          <img src="{{ post.image | relative_url }}">
        </a>
      </div>
      {% endif %}
      <div class="post-entry">
        {{ post.excerpt | strip_html | xml_escape | truncatewords: site.excerpt_length }}
        {% assign excerpt_word_count = post.excerpt | number_of_words %}
        {% if post.content != post.excerpt or excerpt_word_count > site.excerpt_length %}
          <a href="{{ post.url | relative_url }}" class="post-read-more">[Read&nbsp;More]</a>
        {% endif %}
      </div>
    </div>

    {% if post.tags.size > 0 %}
    <div class="blog-tags">
      Tags:
      {% if site.link-tags %}
      {% for tag in post.tags %}
      <a href="{{ '/tags' | relative_url }}#{{- tag -}}">{{- tag -}}</a>
      {% endfor %}
      {% else %}
        {{ post.tags | join: ", " }}
      {% endif %}
    </div>
    {% endif %}

   </article>
  {% endfor %}
</div>

{% if paginator.total_pages > 1 %}
<ul class="pager main-pager">
  {% if paginator.previous_page %}
  <li class="previous">
    <a href="{{ paginator.previous_page_path | relative_url }}">&larr; Newer Posts</a>
  </li>
  {% endif %}
  {% if paginator.next_page %}
  <li class="next">
    <a href="{{ paginator.next_page_path | relative_url }}">Older Posts &rarr;</a>
  </li>
  {% endif %}
</ul>

{% endif %}

