---
layout: page
title: Publications
bigimg:

 - "/img/coverphotos/share-seq-umap.jpg" : "人类肾脏结构的单细胞多组学图谱 Single-cell multimodal atlas for human kidneys (Cell Metabolism 2024)"
 - "/img/coverphotos/sci-seq-injury.jpg" : "近端小管损伤型细胞态 Injured cell states of the proximal tubule (Cell Metabolism 2022；左图被选为NIDDK年度报告唯一插图)"
 - "/img/coverphotos/sci-seq-plin2.jpg" : "PLIN2是一类损伤型肾小管细胞态的脂滴标志物 PLIN2 is a lipid droplet marker in Type 1 Injured PT cells (Cell Metabolism 2022)"
 - "/img/coverphotos/kidney-hic-cutrun.jpg" : "人类肾脏的三维基因组和表观遗传组 The 3D genomics and epigenomics of the human kidney (Scientific Data 2024)"

---


[Google scholar](https://scholar.google.com/citations?user=6EkjrvkAAAAJ&hl=en&oi=ao)

[PubMed](https://www.ncbi.nlm.nih.gov/myncbi/haikuo.li.1/bibliography/public/)

<hr>
## Selected Publications


<strong><u>Li, H.</u></strong>, Li, D., Ledru, N., Xuanyuan, Q., Wu, H., Asthana, A., Byers, L.N., Tullius, S.G., Orlando, G., Waikar, S.S. and Humphreys, B.D., 2024. Transcriptomic, epigenomic, and spatial metabolomic cell profiling redefines regional human kidney anatomy. <strong><u>Cell Metabolism</u></strong>, 36(5), pp.1105-1125. ([DOI](https://doi.org/10.1016/j.cmet.2024.02.015))<br>
→	Featured Content of the <i>Cell Metabolism</i> journal website
<br>

<strong><u>Li, H.</u></strong>, Dixon, E.E., Wu, H. and Humphreys, B.D., 2022. Comprehensive single-cell transcriptional profiling defines shared and unique epithelial injury responses during kidney fibrosis. <strong><u>Cell Metabolism</u></strong>, 34(12), pp.1977-1998. ([DOI](https://doi.org/10.1016/j.cmet.2022.09.026))<br>
→	Research featured by NIH NIDDK annual report ([PDF](https://www.niddk.nih.gov/-/media/Files/Strategic-Plans/2024/NIDDK_RecentAdvances_2024-KUH.pdf))<br>
→	Research Highlights by <i>Nature Reviews Nephrology</i> ([link](https://www.nature.com/articles/s41581-022-00657-x))<br>
→	Research Highlights by <i>Kidney International</i> ([link](https://doi.org/10.1016/j.kint.2023.02.005))
<br>

<strong><u>Li, H.#</u></strong>, Li, D.# and Humphreys, B.D., 2024. Chromatin conformation and histone modification profiling across human kidney anatomic regions. <strong><u>Scientific Data</u></strong>, 11(1), p.797. ([DOI](https://doi.org/10.1038/s41597-024-03648-8))<br>
→	Highlighted in the collection of <i>Data for epigenetics research</i> ([link](https://www.nature.com/collections/fehdjjbjce))
<br>


<hr>
## All Publications

19. Baysoy, A., Tian, X., Zhang, F., Renauer, P., Bai, Z., Shi, H., <strong><u>Li, H.</u></strong>, Tao, B., Yang, M., Enninful, A. and Gao, F., 2024. Spatially Resolved in vivo CRISPR Screen Sequencing via Perturb-DBiT. bioRxiv, pp.2024-11.
<br>

18. Bai, Z., Zhang, D., Gao, Y., Tao, B., Zhang, D., Bao, S., Enninful, A., Wang, Y., <strong><u>Li, H.</u></strong>, Su, G. and Tian, X., Zhang, N., Xiao, Y., Liu, Y., Gerstein, M., Li, M., Xing, Y., Lu, J., Xu, ML. and Fan, R. 2024. Spatially exploring RNA biology in archival formalin-fixed paraffin-embedded tissues. Cell (DOI)
<br>

17. Zhao, C., Rong, K., Liu, P., Kong, K., <strong><u>Li, H.</u></strong>, Zhang, P., Chen, X., Fu, Q. and Wang, X., 2024. Preventing periprosthetic osteolysis in aging populations through lymphatic activation and stem cell-associated secretory phenotype inhibition. Communications Biology, 7(1), p.962. (DOI)
<br>

16. <strong><u>Li, H.</u></strong> and Humphreys, B.D., 2024. Protocol for multimodal profiling of human kidneys with simultaneous high-throughput ATAC and RNA expression with sequencing. STAR Protocols, 5(3), p.103049. (DOI)
<br>

15. <strong><u>Li, H.#</u></strong>, Li, D.# and Humphreys, B.D., 2024. Chromatin conformation and histone modification profiling across human kidney anatomic regions. <strong><u>Scientific Data</u></strong>, 11(1), p.797. ([DOI](https://doi.org/10.1038/s41597-024-03648-8))<br>
→	Highlighted in the collection of <i>Data for epigenetics research</i> ([link](https://www.nature.com/collections/fehdjjbjce))
<br>

14. <strong><u>Li, H.</u></strong> and Humphreys, B.D., 2024. Spatially resolved metabolomic dataset of distinct human kidney anatomic regions. Data in Brief, 54, p.110431. (DOI)
<br>

13. <strong><u>Li, H.</u></strong>, Li, D., Ledru, N., Xuanyuan, Q., Wu, H., Asthana, A., Byers, L.N., Tullius, S.G., Orlando, G., Waikar, S.S. and Humphreys, B.D., 2024. Transcriptomic, epigenomic, and spatial metabolomic cell profiling redefines regional human kidney anatomy. <strong><u>Cell Metabolism</u></strong>, 36(5), pp.1105-1125. ([DOI](https://doi.org/10.1016/j.cmet.2024.02.015))
<br>
→	Featured Content of the <i>Cell Metabolism</i> journal website
<br>

12. <strong><u>Li, H.</u></strong> and Humphreys, B.D., 2024. Multimodal characterization of sexual dimorphism in the mammalian kidney. Kidney International, 105(4), pp.653-655. (DOI)
<br>

11. <strong><u>Li, H.</u></strong> and Humphreys, B.D., 2024. Targeting de novo lipogenesis to mitigate kidney disease. The Journal of Clinical Investigation, 134(4). (DOI)
<br>

10. Wilson, P.C., Verma, A., Yoshimura, Y., Muto, Y., <strong><u>Li, H.</u></strong>, Malvin, N.P., Dixon, E.E. and Humphreys, B.D., 2024. Mosaic loss of Y chromosome is associated with aging and epithelial injury in chronic kidney disease. Genome biology, 25(1), p.36. (DOI)
<br>

9. Wang, J.#, Ku, X.#, Ma, Q.#, <strong><u>Li, H.</u></strong>#, Huang, S.#, Mao, L., Yu, F., Jin, J. and Yan, W., 2024. Hsa_circ_0007099 and PIP4K2A coexpressed in diffuse large B-cell lymphoma with clinical significance. Genes & Diseases, 11(4), p.101056. (DOI)
<br>

8. Ku, X.#, Wang, J.#, <strong><u>Li, H.</u></strong>#, Meng, C., Yu, F., Yu, W., Li, Z., Zhou, Z., Zhang, C., Hua, Y. and Yan, W., 2023. Proteomic portrait of human lymphoma reveals protein molecular fingerprint of disease specific subtypes and progression. Phenomics, 3(2), pp.148-166. (DOI)
<br>

7. Cheng, Z. et al. (special journal issue; author list alphabetically ordered) (2023). RNA-seq: questions and answers. STAR Protocols. (DOI)<br>

6. <strong><u>Li, H.</u></strong> and Humphreys, B.D., 2022. Mouse kidney nuclear isolation and library preparation for single-cell combinatorial indexing RNA sequencing. STAR Protocols, 3(4), p.101904. (DOI)
<br>

5. <strong><u>Li, H.</u></strong>, Dixon, E.E., Wu, H. and Humphreys, B.D., 2022. Comprehensive single-cell transcriptional profiling defines shared and unique epithelial injury responses during kidney fibrosis. <strong><u>Cell Metabolism</u></strong>, 34(12), pp.1977-1998. ([DOI](https://doi.org/10.1016/j.cmet.2022.09.026))<br>

→	Research featured by NIH NIDDK annual report ([PDF](https://www.niddk.nih.gov/-/media/Files/Strategic-Plans/2024/NIDDK_RecentAdvances_2024-KUH.pdf))<br>

→	Research Highlights by <i>Nature Reviews Nephrology</i> ([link](https://www.nature.com/articles/s41581-022-00657-x))<br>

→	Research Highlights by <i>Kidney International</i> ([link](https://doi.org/10.1016/j.kint.2023.02.005))
<br>

4. Muto, Y.#, <strong><u>Li, H.</u></strong># and Humphreys, B.D., 2022. Single Cell Transcriptomics. Innovations in Nephrology: Breakthrough Technologies in Kidney Disease Care (pp. 87-102). Cham: Springer International Publishing. (DOI)
→	Chapter has been translated into German (DOI)
<br>

3. <strong><u>Li, H.</u></strong> and Humphreys, B.D., 2022. New functions for basophils identified in kidney fibrosis. Nature Immunology, 23(6), pp.824-825. (DOI)
<br>

2. <strong><u>Li, H.</u></strong> and Humphreys, B.D., 2021. Single cell technologies: Beyond microfluidics. Kidney360, 2(7), pp.1196-1204. (DOI)
<br>

1. <strong><u>Li, H.</u></strong> and Humphreys, B.D., 2020. Surveying the human single-cell landscape. Kidney International, 98(6), pp.1385-1387. (DOI)
<br>Targeting
